prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/HI/HI-receiver-transform.yml

```diff
@@ -0,0 +1,89 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+elements:
+
+  # XTN
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: hi-receiving-application
+    condition: true
+    value: [ '"HIDOH"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: hi-receiving-facility
+    condition: true
+    value: [ '"HIDOH"' ]
+    hl7Spec: [ 'MSH-6-1' ]
+
+  - name: message_profile_id
```

Comment:
I think the transforms from here through software-id have been added to the simple report sender transform and no longer needed here.